markup/goldmark/convert.go

```diff
@@ -102,9 +102,26 @@ func newMarkdown(pcfg converter.ProviderConfig) goldmark.Markdown {
 	if rendererOptions != nil {
 		copy(tocRendererOptions, rendererOptions)
 	}
+
 	tocRendererOptions = append(tocRendererOptions,
 		renderer.WithNodeRenderers(util.Prioritized(extension.NewStrikethroughHTMLRenderer(), 500)),
-		renderer.WithNodeRenderers(util.Prioritized(emoji.NewHTMLRenderer(), 200)))
+		renderer.WithNodeRenderers(util.Prioritized(emoji.NewHTMLRenderer(), 200)),
+	)
+
+	inlineTags := []extras.InlineTag{
+		extras.DeleteTag,
+		extras.InsertTag,
+		extras.MarkTag,
+		extras.SubscriptTag,
+		extras.SuperscriptTag,
```

Comment:
This needs to respect  the configuration, ans since that information is known to the extras package, we could possibly create a `ConfiguredTags` method or something, and then change the signature below to return something ... more:

>func New(config Config) goldmark.Extender {